DataDog/datadog-agent
# [NET-999][system-probe] Add runtime compilation telemetry

### What does this PR do?
Adds telemetry related to runtime compilation to the system probe expvar endpoint

### Motivation

[NET-999](https://datadoghq.atlassian.net/browse/NET-999)

### Additional Notes

### Describe your test plan
1) Add the following to your system_probe.yaml file:
```
system_probe_config:
  enable_runtime_compiler: true
```
2) Run the system probe
3) Check that `curl -s --unix-socket /opt/datadog-agent/run/sysprobe.sock http://unix/connections?client_id=1 | jq .compilationTelemetryByAsset` returns the following stats:
```
{
  "conntrack": {
    "runtimeCompilationEnabled": true,
    "runtimeCompilationResult": [string describing result of compilation],,
    "runtimeCompilationDuration": [n > 0]
  },
 "tracer": { [same idea] }
}
```
